samples/go/photo-index/main.go

```diff
@@ -27,21 +29,13 @@ func index() (win bool) {
 	var dbStr = flag.String("db", "", "input database spec")
 	var outDSStr = flag.String("out-ds", "", "output dataset to write to - if empty, defaults to input dataset")
 	var parallelism = flag.Int("parallelism", 16, "number of parallel goroutines to search")
+	verbose.RegisterVerboseFlags(flag.CommandLine)
 
 	flag.Usage = usage
 	flag.Parse(false)
 
-	if flag.NArg() == 0 {
-		flag.Usage()
-		return
-	}
-
-	if flag.NArg() == 0 {
```

Comment:
My mistake. I misread the code. I restored one check. (The second was dead code).